sdk/schemaregistry/azure-data-schemaregistry-avro/src/main/java/com/azure/data/schemaregistry/avro/AvroSchemaUtils.java

```diff
@@ -3,8 +3,8 @@
 
```

Comment:
I may collapse the two util classes because they can live in the serializer class and I don't see a use elsewhere for them.